### DISCUSSION TITLE
After update to version 3.9.21 (30 jan) progress bar showing only

### DISCUSSION THREAD

**Discussion by MonkAlex** [piroor/treestyletab#3439] [Q&A] [answered]
Today extension updated and now its look like this:
![image](https://github.com/piroor/treestyletab/assets/6784764/4517c000-3f9e-4844-b599-db391dd10957)

Test on 2 profiles and in private mod too.
FF version 122.

**Comment by MonkAlex** [comment]
When open moz-extension://cb49380f-5d01-444e-8210-fa0dad295bf7/sidebar/sidebar.html as page in console only one row:
`managed storage is not provided [Configs.js:310:27](moz-extension://cb49380f-5d01-444e-8210-fa0dad295bf7/extlib/Configs.js)`

**Comment by MonkAlex** [comment, marked as answer]
Okay, TST work when i enable " Read and modify bookmarks " permission. 
But permission in "Optional" block and i'm not sure, why its required now.

@piroor

**Reply by piroor** [reply]
This issue is tracked at #3440 and this is an effective workaround until 3.9.22 is published.
